### DISCUSSION TITLE
Scorpion vs. White Ranger

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#460] [Vídeos]
Post de referência: https://www.vish.blog.br/videos/scorpion-vs-white-ranger/

**Comment by one-eyed-willy** [comment]
Me deu até um coiso aqui quando ele tirou o capacete

**Comment by angellisy** [comment]
MUITO melhor que o último filme do MK.

**Comment by brunoaugusto** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [quarta-feira, 13 de abril de 2022](<https://www.vish.blog.br/videos/scorpion-vs-white-ranger/#comment_1098536081>):

> MUITO melhor que o último filme do MK.

Pois é, né? Eu fiquei na maior expectativa e o filme foi decepcionante
